.github/workflows/ci.yml

```diff
@@ -129,6 +129,10 @@ jobs:
           version: latest
           skip-go-installation: true
           skip-pkg-cache: true
+      - name: Custom Lint
+        run: |
+          go run linter/koanf/koanf.go linter/koanf/handlers.go ./...
+          go run linter/pointercheck/pointer.go ./...
```

Comment:
Could we do `go run ./linter/pointercheck ./...` to avoid listing the go files in the linter? Same for the koanf linter of course.

Reply:
Maybe I'm missing something obvious but it complains with `package linter/pointercheck is not in GOROOT`. 

If I change it to `go run github.com/offchainlabs/nitro/linter/linter/pointercheck ./...` I think it will not run changes in linter in the same pr that changes it.

Also tried just `go run linter/linter/* ./...` and then it complains about not being able to execute run a file.

Reply:
Locally for me it works with `go run ./linter/pointercheck ./...`, is there something different about the CI environment?

Reply:
That's exactly what I was missing, with `./` prefix it works fine. Thanks Tristan!